src/main/java/net/minecraftforge/fml/common/Loader.java

```diff
@@ -680,78 +682,110 @@ public ModClassLoader getModClassLoader()
         return modClassLoader;
     }
 
-    public void computeDependencies(String dependencyString, Set<ArtifactVersion> requirements, List<ArtifactVersion> dependencies, List<ArtifactVersion> dependants)
+    public static void computeDependencies(String dependencyString, Set<ArtifactVersion> requirements, List<ArtifactVersion> dependencies, List<ArtifactVersion> dependants)
     {
         if (dependencyString == null || dependencyString.length() == 0)
```

Comment:
Ok. I will wait for more important remarks anyways, and if there are any, i may change some of these.